core/p2p/core.go

```diff
@@ -224,6 +221,15 @@ func provide(c *dig.Container) {
 }
 
 func configure() {
+
+	// make sure nobody copies around the peer store since it contains the
+	// private key of the node
+	log.Infof("never share your %s folder as it contains your node's private key!", deps.NodeConfig.String(CfgP2PPeerStorePath))
+
+	if !peerStoreIsNew {
+		log.Infof("retrieving existing peer identity from %s", path.Join(deps.NodeConfig.String(CfgP2PPeerStorePath), pubKeyFileName))
```

Comment:
Since this is now printed out on configure(), you can change it to be "retrieved existing peer identity..", since the initialization of the libp2p host already happened.